cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -2396,6 +2412,108 @@ func TestFilterOutYoungPods(t *testing.T) {
 	}
 }
 
+func TestStaticAutoscalerRunOnceInvokesScaleDownStatusProcessorNoCandidates(t *testing.T) {
```

Comment:
Could we remove the "NoCandidates" part from the name, and add test cases for both when there are and aren't candidates?

Reply:
Totally, makes sense, added 3 more test cases to cover this behavior